Notifications/notifications_actions_customization.md

```diff
@@ -12,48 +12,51 @@ This document is a starting point for engaging the community and standards bodie
 
 ## Goals
 
-Propose an extension to the Notifications API to allow more customization for VoIP web apps. 
+Propose an extension to the Notifications API to allow more customization for VoIP Progressive Web Apps (PWAs). 
 
 ## Non-Goals
 
 Allow general web contents to arbitrarily change action button colors and sound sources without restrictions from the underly platforms.
 
 ## Use Cases 
 
-- A video conferencing website can produce its call notification pop-up with a green "accept" button and red "decline" button.
-- Web apps will be capable to change the notification action button colors if they feel like this will improve it UX.
+- A video conferencing PWA can produce its call notification pop-up with a green "accept" button and red "decline" button.
+- Web apps will be capable to change the notification action button colors if they feel like this will improve its UX.
 
 ## Proposed Solution
 
-We propose an extension to the current Notifications API where the `NotificationAction` dictionary would have a new `style` variable of type `NotificationActionStyle`, which is an enum with 3 values: 
-- `"default"`,
-- `"success"`, and
-- `"critical"`.
-
-These styles are platform-dependent but should be distinct from each other to allow the user to easily identify its options. For example, the `"success"` action button could have a green theme - e.g., have green background color or have a green font over the default background - while the `"critical"` could be red-themed, but it is up to the platform to define the style. If the selected style is not supported by the platform, it should fallback to `"default"`
+We propose the creation of a new property `scenario` in the `NotificationOptions` dictionary of type `NotificationScenario`, that is an `enum` with 2 values: `"default"` and `"calling"`. The color treatment and ringtone capabilities will only be available in the `"calling"` scenario.
 
 ```javascript
-enum NotificationActionStyle {
+enum NotificationScenario {
   "default",
-  "success",
-  "critical"
+  "calling"
 }
 
-dictionary NotificationAction {
-  required DOMString action;
-  required DOMString title;
-  USVString icon;
-  NotificationActionStyle style = "default";
-};
+dictionary NotificationOptions {
+  NotificationScenario scenario = "default";
+  ...
+}
 ```
 
-Similarly, to support this scenario we also propose the creation of a new NotificationSoundType enum and the inclusion of a `sound` attribute to the Notification interface and a correspondent property to the NotificationOptions dictionary:
+When the scenario is `"calling"`, the notification will always have a default dismiss button, which must occupy the rightmost action button position. This way, when a `"calling"` notification is displayed to the user, we can guarantee that he or she will always be capable to dismiss it. 
```

Comment:
I'm not sure we have to dictate the position and layout of the dismiss button.  For example, instead of on the right, a browser could choose to have it occupy an entire bottom row.  We can leave it up to browser implementers to decide how to show the dismiss button.

Reply:
Done. Thanks!